Editor/Resources/EditorWindow/Pages/HelpAndDocumentationPage.uxml

```diff
@@ -1,33 +1,34 @@
 <!-- Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. -->
 <!-- SPDX-License-Identifier: Apache-2.0 -->
 
-<ui:UXML xmlns:ui="UnityEngine.UIElements">
+<ui:UXML xmlns:ui="UnityEngine.UIElements" xmlns:gl="AmazonGameLift.Editor">
     <ui:Style src="./HelpAndDocumentationPage.uss"/>
     <ui:VisualElement class="separator separator--large">
         <ui:VisualElement class="separator">
             <ui:Label name="HelpPageTitle" class="page__title" text="Help &amp; Documentation"/>
             <ui:Label name="HelpPageDescription" text="Learn how to integrate and deploy games for hosting on GameLift. The GameLift service manages game server deployment, operation, and scaling.&#10;You can deploy custom-built game servers or use GameLift Realtime Servers to configure lightweight game servers for your game."/>
+            <gl:InfoLinks/>
         </ui:VisualElement>
         <ui:VisualElement class="divider"/>
         <ui:VisualElement class="separator separator--horizontal">
-            <ui:VisualElement class="card card--small">
+            <ui:VisualElement name="HelpPageReportIssueLink" class="card card--small">
                 <ui:Image name="HelpPageReportIssueIcon" class="card__icon"/>
-                <ui:Label name="HelpPageReportIssueLink" class="card__text" text="Report Issue   \uf08e"/>
+                <ui:Label name="HelpPageReportIssueLinkText" class="card__text" text="Report Issue"/>
```

Comment:
All Link elements in the project end with `Link`. Refer `Strings.cs`. Can you call the parent `Link` and this element `Label`? Also please apply this to all `Link` elements

Reply:
I have changed it for the parent to be not named and the text have correct names so it can get text from text provider.